### ISSUE TITLE
There is no authenticated way to download installers.

### ISSUE THREAD

**Issue by BalzGuenat** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#1413]
The site www.googleplaymusicdesktopplayer.com is not accessible securely via https. This means there is no way to verify the authenticity of the downloaded installers which users are meant to run. This is a security risk. Please fix this by getting a certificate for the site/domain and using https, ideally by default.

**Comment by MarshallOfSound** [comment]
If you do a trace the installers are not fetched from `www.googleplaymusicplayer.com`.

They are fetched from a SSL secured GitHub artifact domain and proxied to S3.

**Comment by BalzGuenat** [comment]
That doesn't help if the site linking to the artifact isn't secure. If someone can spoof the website, they can put any link they want there.

**Comment by MarshallOfSound** [comment]
Getting an SSL cert for `www.googleplaymusicdesktopplayer.com` is an absolute nightmare because "google" is in the name.

TLDR, if you want a secure response use `https://gpmdp.xyz`

**Comment by BalzGuenat** [comment]
Alright, that seems reasonable.

**Comment by MarshallOfSound** [comment]
FWIW as well, I managed to get a Let's Encrypt cert for the core site now as well.  It means not using Cloudflare though so we shall see how it goes

https://www.googleplaymusicdesktopplayer.com/
